### ISSUE TITLE
Open 'Try in REPL' from the example sections in the new tab

### ISSUE THREAD

**Issue by ivanzusko** [ramda/ramda#2217]
Hello, everybody.

In my humble opinion, it's kinda annoying when you are on [docs](http://ramda/docs/) page and click on the `Try in REPL` it's opening REPL in the same tab. After all you need to press backspace to return back to the docs page. And more changes in REPL you did - more times you need to press backspace.

I'd like to make PR, but can't find the place where this code is living:
```javascript
function tryInREPL(event) {
    if (!event.target.matches('.try-repl')) {
      return;
    }
    var version = event.target.dataset && event.target.dataset.ramdaVersion;
    var versionParam = version ? '?v=' + version : '';
    var code = event.target.nextElementSibling.textContent;
    var encoded = fixedEncodeURIComponent(code);
    location.assign(location.origin + '/repl/' +
      versionParam + '#;' + encoded);
  }
```

I've looked everything in repo files but didn't manage to find :disappointed: 

Basically, what we need to do is replace `location.assign` with `window.open`. As I'm guessing, this lives somewhere in here maybe `/lib/docs/public/`, but this path is in **.gitignore**.
Would you mind to reference me to the target file where `tryInREPL` lives?

Cheers!

**Comment by ivanzusko** [comment]
I've done same manipulation in [here](https://github.com/ivanzusko/ramda-docs-chrome-extension/commit/0ca6d640ff9aed646134c6f0c93b845eb4bbec64) and it works perfectly for Chrome extension.

**Comment by buzzdecafe** [comment]
Hi -- please open this issue/PR in the docs repo, ramda/ramda.github.io

Thanks

**Comment by ivanzusko** [comment]
@buzzdecafe thank you! Done
